metadata-ingestion/src/datahub/utilities/sql_lineage_parser_impl.py

```diff
@@ -32,18 +35,59 @@ def __init__(self, sql_query: str) -> None:
         if "lateral flatten" in sql_query:
             sql_query = sql_query[: sql_query.find("lateral flatten")]
 
+        # BigQuery can use # as a comment symbol and that may break the parsing
+        # if the comment starts without a space sign after #
+        # and the comment is written inside SQL DDL statement (e.g., after CREATE...AS)
+        sql_query = re.sub(r"#([^ ])", r"# \1", sql_query, flags=re.IGNORECASE)
+
+        # Wrap calls to field '<table_alias>.from' in ` so it would not be taken as a reserved keyword
+        sql_query = re.sub(r"(\w*\.from)", r"`\1`", sql_query, flags=re.IGNORECASE)
+
+        # Apply sqlparser formatting to get rid of comments and reindent keywords
```

Comment:
@treff7es Will removing comments break anything? I remember you using comments for something.

Reply:
No, that is a different thing. Feel free to remove it.